internal/errors/errors.go

```diff
@@ -263,10 +263,22 @@ func Join(errs ...error) error {
 	var e *joinError
 	switch x := errs[0].(type) {
 	case *joinError:
-		e = x
+		if x != nil {
```

Comment:
```suggestion
		if x != nil && len(x.errs) != 0 {
```

Reply:
Thank you for catching it! I apply your suggestion!